sdk/search/Azure.Search.Documents/tests/Utilities/SearchTestBase.cs

```diff
@@ -73,9 +68,9 @@ public SearchClientOptions GetSearchClientOptions(SearchClientOptions options =
             options ??= new SearchClientOptions(ServiceVersion);
             options.Diagnostics.IsLoggingEnabled = true;
             options.Retry.Mode = RetryMode.Exponential;
-            options.Retry.MaxRetries = MaxRetries;
-            options.Retry.Delay = TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0.01 : 0.5);
-            options.Retry.MaxDelay = TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0.1 : 10);
+            options.Retry.MaxRetries = 10;
+            options.Retry.Delay = TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0.01 : 1);
```

Comment:
Wonder if this should go into InstrumentOptions()...

Reply:
I think it would be a good idea because folks probably don't notice when their recordings include retries.  Any chance someone's relying on the existing behavior though?